easybuild/easyconfigs/p/ParaView/ParaView-5.9.1.eb

```
@@ -0,0 +1,29 @@
+easyblock = 'Binary'
+
+name = 'ParaView'
+version = '5.9.1'
+
+homepage = "https://www.paraview.org"
+description = "ParaView is a scientific parallel visualizer."
+
+toolchain = SYSTEM
+
+_download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=binary&os=Linux&downloadFile='
+source_urls = ['https://www.paraview.org/paraview-downloads/%s' % _download_suffix]
+sources = ["ParaView-%(version)s-MPI-Linux-Python3.8-64bit.tar.gz"]
+checksums = ['7011b0ebb24ed17b33891af255ed55e90b7cf17834851afc55c908e00419a4a3']
+
+extract_sources = True
+
+sanity_check_paths = {
+    'files': ['bin/paraview'],
+    'dirs': [],
+}
+
+prepend_to_path = ['bin']
+
+modextrapaths = {
+    'LD_LIBRARY_PATH': ['lib', 'lib/mesa'],
+}
```

```suggestion
modextrapaths = {
    'LD_LIBRARY_PATH': ['lib/mesa'],
}
```
Wouldn't `bin` and `lib` automatically be added to `$PATH` and `$LD_LIBRARY_PATH` respectively?

---

Yes, you're correct. This wasn't the case with another build the other day but I appear to have migrated the same thinking to ParaView!